setup.cfg

```diff
@@ -14,3 +14,4 @@ parentdir_prefix = great_expectations-
 [flake8]
 max-line-length = 200
 exclude = tests/*,docs/*
+per-file-ignores = */__init__.py: F401 # Allow unused imports in __init__'s
```

Comment:
F401 is the warning for unused imports - I've decided to silence this globally since we never actually use the imported class/function in the init.